db/migrate/20220223095704_remove_col3_from_miq_widget_set.rb

```diff
@@ -0,0 +1,31 @@
+class RemoveCol3FromMiqWidgetSet < ActiveRecord::Migration[6.0]
+  class MiqWidgetSet < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.table_name = "miq_sets"
+    serialize :set_data
+  end
+
+  def up
+    say_with_time("Moving col3 widgets to col2 and removing col3") do
+      MiqWidgetSet.all.each do |item|
+        data = item.set_data
+        if data
+          data[:col2] = ((data[:col2] || []) + (data[:col3] || [])).uniq.compact
```

Comment:
Just noticed the `uniq.compact`.  Can you possibly add a test that exercises this?  Perhaps an id that is in both col2 and col3?